pkg/build/registry/buildconfig/webhook.go

```diff
@@ -72,9 +73,13 @@ func (w *WebHook) ServeHTTP(writer http.ResponseWriter, req *http.Request, ctx k
 		Revision:    revision,
 		Env:         envvars,
 	}
-	if _, err := w.instantiator.Instantiate(config.Namespace, request); err != nil {
+	newBuild, err := w.instantiator.Instantiate(config.Namespace, request)
+	if err != nil {
 		return errors.NewInternalError(fmt.Errorf("could not generate a build: %v", err))
 	}
+	// Send back the build name so that the client can alert the user.
+	newBuildJson, _ := json.Marshal(newBuild)
```

Comment:
don't throw away errors.  I think we should return an internal error, though that my be confusing since the build still got created.  @openshift/api-review ?